app/bundles/ReportBundle/Form/Type/DynamicFiltersType.php

```diff
@@ -66,16 +68,22 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                         $args['html5']  = false;
                         $args['format'] = 'y-MM-dd HH:mm:ss';
                         $args['attr']['class'] .= ' datetimepicker';
+                        if (!isset($options['data'][$definition['alias']]) && (empty($filter['value']) || '' === $filter['value'])) {
+                        }
                         break;
                     case 'multiselect':
                         $args['multiple']         = true;
                         // no break
                     case 'select':
                         $type            = ChoiceType::class;
                         $args['choices'] = array_flip($definition['list']);
+                        if (!isset($options['data'][$definition['alias']]) && (empty($filter['value']) || '' === $filter['value'])) {
+                        }
```

Comment:
```suggestion

```
The conditional block is empty. It should be removed if unnecessary or completed with logic.